app/mailers/invisiblize_mailer.rb

```diff
@@ -0,0 +1,25 @@
+class InvisiblizeMailer < ApplicationMailer
+  def to_developer
+    @notification = params[:record].to_notification
+    recipient = params[:recipient]
+
+    @developer = @notification.developer
+
+    mail(
+      to: recipient.email,
+      subject: @notification.title
+    )
+  end
+
+  def to_business
```

Comment:
```suggestion
  def business
```

Reply:
This will break the notification unfortunately, see above